builtin/providers/aws/resource_aws_network_interface.go

```diff
@@ -46,6 +46,12 @@ func resourceAwsNetworkInterface() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"source_dest_check": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
We need to add `Default: true,` here, or otherwise any user who omits the `source_dest_check` attribute will send a `false` value to the API